packages/adapter-cloudflare-workers/index.js

```diff
@@ -140,6 +141,30 @@ export default function ({ config = 'wrangler.toml' } = {}) {
 			const bucket_dir = `${site.bucket}${builder.config.kit.paths.base}`;
 			builder.writeClient(bucket_dir);
 			builder.writePrerendered(bucket_dir);
+		},
+
+		async emulate() {
+			const proxy = await getBindingsProxy();
```

Comment:
FYI I was just messing around with this locally using a `hooks.server.ts` file. One thing I noticed is when I forgot to `proxy.dispose()`, a lot of zombie `containerd` processes were left orphaned. Just noting that here in case it is helpful/needed to complete this implementation.